contracts/src/LightClient.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+import { BN254 } from "bn254/BN254.sol";
+import { IPlonkVerifier } from "./interfaces/IPlonkVerifier.sol";
+import { IStakeTable } from "./interfaces/IStakeTable.sol";
+import { PlonkVerifier } from "./libraries/PlonkVerifier.sol";
+
+// TODO: replace these, for now WIP and test only
+import { VkTest } from "../test/stubs/Transfer1In2Out24DepthVk.sol";
+
+/// @notice A light client for HotShot consensus. Keeping track of its finalized states in safe,
+/// authenticated ways.
+contract LightClient {
+    // === Constants ===
+    //
+    /// @notice System parameter: number of blocks per epoch
+    uint32 public constant BLOCKS_PER_EPOCH = 20_000;
```

Comment:
Maybe we should change this to `immutable` and set it in the constructor. We might want to play around with different epoch lengths and it would be nice to do this without changing the code of the contract.

Reply:
good suggestion, done in [29392af](https://github.com/EspressoSystems/espresso-sequencer/pull/754/commits/29392afd5c14a3940ee1ea863c63ba3660e99380)